GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/Content/SectionSynchronizer.cs

```diff
@@ -52,12 +55,13 @@ private IEnumerable<TestRailSection> GetSectionsTree(ulong projectId, ulong? sui
         /// Gets or creates TestRail section Id for selected .feature file
         /// </summary>
         /// <param name="path">Path to .feature file</param>
-        /// <param name="suiteId">TestRail suite Id</param>
-        /// <param name="projectId">TestRail project Id</param>
         /// <returns></returns>
         /// <exception cref="ArgumentNullException"></exception>
-        public ulong GetOrCreateSectionId(string path, ulong suiteId, ulong projectId)
```

Comment:
As we discussed before, it is logical to leave **suiteId** and **projectId** as parameters, since we are already getting them in the calling method **TestRailSynchronizer.Sync(List<IFeatureFile> featureFiles)**

Reply:
After my changes, `TestRailSynchronizer` knows nothing about a suitId, a projectId, and a templateId, because no action is required here with this info.